### ISSUE TITLE
[Image] Alt text stays mandatory field even when 'decorative' flag is set on first save

### ISSUE THREAD

**Issue by msagolj** [adobe/aem-core-wcm-components#364]
## Bug Report

**Current Behavior**
- Add an image component (can be a proxy) to the page (either drag'n'drop image or image component)
- Open edit dialog
- Edit properties, set the 'Image is decorative' flag
- save
- reopen dialog
- try to save again -> fails, it expects 'alt text' to be set as mandatory field, even though the 'Image is decorative' flag should disable it completely.

**Expected behavior/code**
- Add image component
- Open Edit dialog
- Edit properties, set the 'Image is decorative' flag
- save
- reopen dialog
- try to save again ->works 

**Environment**
- AEM : all
- Core Components : 2.0.8, 2.2.0 , probably all versions including the alt text field

(CQ-4256860)

**Comment by richardhand** [comment]
This was fixed with the merge of #366 and is on master. It will be available in the upcoming [2.2.2 release](https://github.com/adobe/aem-core-wcm-components/milestone/9).
